ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.html

```diff
@@ -2,7 +2,7 @@
   <table class="table table-striped Width--100">
     <thead>
     <tr>
-      <th>
+      <th *ngIf="!canNotSubmitOrder()">
         Select
```

Comment:
I removed the selection check boxes because they are not useful anymore in a view only permission